README.md

```diff
@@ -1,26 +1,34 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/robot-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Revit_Toolkit/Revit_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=99&branchName=master)
```

Comment:
```suggestion
[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/d1y1h3ywkch3fd06/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/revit-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Revit_Toolkit/Revit_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=99&branchName=master)
```

Reply:
Hey @pawelbaran - think the above should work for you.
Sorry - yes - should have highlighed this before - not easy access this links. @IsakNaslundBh @alelom FYI for when you are doing your READMEs - let me @FraserGreenroyd know to help with badges